### ISSUE TITLE
No ability to disable automatic polling for new resources

### ISSUE THREAD

**Issue by jefftrimm** [aresok/go-maven-poller#4]
In certain pipelines we want to be able to retrieve the latest metadata about an artifact in maven when the pipeline runs, but don't want the pipeline to be triggered by an artifact change.  Similar to the source control pollers, the plugin needs a way to to select/deselect "Automatically Poll Repository for Changes".

**Comment by arvindsv** [comment]
Would making the first stage "Manual" help you achieve what you're trying to?

**Comment by jefftrimm** [comment]
No, we want the pipeline to be triggered only by an upstream pipeline stage.
